.github/workflows/test.yml

```diff
@@ -94,4 +94,4 @@ jobs:
           PYTHONPATH: "$PWD/e2e"
         run: |
           cd e2e
-          python -m pytest .
+          python -m pytest . -k 'not cugraph'
```

Comment:
We should fix this, right?

Reply:
We do not have the machine set up to run this kind of tests on the `Actions` at the moment